lib/validation.rb

```diff
@@ -0,0 +1,23 @@
+module Validation
+  def ensure_format(key)
+    if key =~ /\A(GET|PUT|POST|DELETE|HEAD)\s\/[a-z0-9_]+\z/
+      true
+    else
+      false
+    end
+  end
+
+  def ensure_url(value)
+    unless value =~ URI::regexp
```

Comment:
__[rubocop]__
* Favor modifier `unless` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.
* Do not use `::` for method calls.